src/lib/createables.ts

```diff
@@ -675,6 +676,26 @@ const Createables: Createable[] = [
 			"Ava's assembler": 1
 		}),
 		QPRequired: 205
+	},
+	{
+		name: 'Toxic blowpipe (empty)',
+		inputItems: {
+			[itemID('Toxic blowpipe')]: 1
+		},
+		outputItems: {
+			[itemID('Toxic blowpipe (empty)')]: 1,
+			[itemID(`Zulrah's Scales`)]: 25000
+		}
+	},
+	{
+		name: 'Toxic blowpipe',
+		inputItems: {
+			[itemID('Toxic blowpipe (empty)')]: 1,
+			[itemID(`Zulrah's Scales`)]: 25000
+		},
+		outputItems: {
```

Comment:
i think we should remove this for now... it wont have a use immediately anyway, and i need to decide if ill code an actual blowpipe (and ammo) system for consuming amo